docs/sql-pyspark-pandas-with-arrow.md

```diff
@@ -166,6 +166,12 @@ different than a Pandas timestamp. It is recommended to use Pandas time series f
 working with timestamps in `pandas_udf`s to get the best performance, see
 [here](https://pandas.pydata.org/pandas-docs/stable/timeseries.html) for details.
 
+### Recommended Pandas and PyArrow Versions
+
+For usage with pyspark.sql, the supported versions of Pandas is 0.19.2 and PyArrow is 0.8.0. Higher
```

Comment:
BTW, I forgot the exact versions of the `branch-2.4` PR builders in AmpLab Jenkins. These are the community testing version. Did I understand correctly?

Reply:
Yeah, i believe so.